src/Backend/Synapse/Helper/BackendHelper.php

```diff
@@ -10,4 +10,9 @@ public static function generateTempTableName(): string
     {
         return '#__temp_' . str_replace('.', '_', uniqid('csvimport', true));
     }
+
```

Comment:
mohl bys tam dat malou poznamku proc to potrebujeme. predpokladam, ze to je vlastne docasna vec, nez zjistime, ze ten problem je fakt na synapse.

Reply:
jo klidně přidám, dočasná věc to není ničemu nevadí když to tam tak ostane.